keyboards/ocean/sus/readme.md

```diff
@@ -0,0 +1,20 @@
+# SUS Board
+
+![SUS](https://user-images.githubusercontent.com/30220306/126883340-47401d95-2095-4e55-ba5c-4ae7acaa430f.png)
+
+
+A Macropad like a amogus
+
+* Keyboard Maintainer: Ocean
+* Hardware Supported: Sus Board, Pro Micro
+* Hardware Availability: Private GB
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to put the keyboard into bootloader mode, to flash new firmware?

Reply:
@drashna done :)